build-system/server/app-utils.js

```diff
@@ -80,6 +82,31 @@ const isRtvMode = (serveMode) => {
   return /^\d{15}$/.test(serveMode);
 };
 
+const tsModule = Object.create(null);
+
+/**
+ * @param {string} path
+ * @return {*}
+ */
+function requireTsSync(path) {
```

Comment:
@samouri I should be using `TransformCache` here, but before I do so: do you know a more appropriate place to put this function? should I use an existing one?

Reply:
Now using the same `dist` file locations as `buildNewServer`, so this is moot.